bycycle/burst.py

```diff
@@ -5,67 +5,80 @@
 
 import numpy as np
 import matplotlib.pyplot as plt
-from bycycle.filt import amp_by_time
 from scipy.stats import zscore
 import pandas as pd
+from bycycle.filt import amp_by_time
 
-pd.options.mode.chained_assignment = None
 
+pd.options.mode.chained_assignment = None
 
 def detect_bursts_cycles(df, x, amplitude_fraction_threshold=0,
                          amplitude_consistency_threshold=.5,
                          period_consistency_threshold=.5,
                          monotonicity_threshold=.8,
                          N_cycles_min=3):
-    """
-    Compute consistency between cycles and determine which are truly oscillating
+    """Compute consistency between cycles and determine which are truly oscillating.
 
     Parameters
     ----------
     df : pandas DataFrame
-        dataframe of waveform features for individual cycles, trough-centered
+        Dataframe of waveform features for individual cycles, trough-centered.
     x : 1d array
-        trace used to compute monotonicity
+        Trace used to compute monotonicity.
     amplitude_fraction_threshold : float (0 to 1)
-        the minimum normalized amplitude a cycle must have
-        in order to be considered in an oscillation.
-        0 = the minimum amplitude across all cycles
-        .5 = the median amplitude across all cycles
-        1 = the maximum amplitude across all cycles
+        The minimum normalized amplitude a cycle must have in order to be considered in an
+        oscillation.
+        ..
```

Comment:
If you check how these get rendered in the API listing, these lists don't render well (they get popped out, and made big, like sphinx tends to do if it doesn't like the layout). I think if you put these lists on the same tab line, with a space above, they will render better (for example, like the list in `plot_burst_detect_params`.

Reply:
This should be fixed now.